OpenEmu/pt.lproj/InfoPlist.strings

```diff
@@ -24,12 +24,12 @@
   SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-"Archived Game" = "Archived Game"; // TODO
+"Archived Game" = "Jogo arquivado";
 
-"GameCorePlugin" = "GameCorePlugin"; // TODO
+"GameCorePlugin" = "PluginDoNucleo"; // TODO NOT SURE HOW THIS SHOULD BE TRANSLATED, CONTEXT?
 
-"OpenEmu Save State" = "OpenEmu Save State"; // TODO
+"OpenEmu Save State" = "Salvamento OpenEmu";
 
-"System Plugin" = "System Plugin"; // TODO
+"System Plugin" = "Plugin do sistema";
 
-"VideoFilterPlugin" = "VideoFilterPlugin"; // TODO
+"VideoFilterPlugin" = "PluginFiltroDeVideo"; // TODO NOT SURE HOW THIS SHOULD BE TRANSLATED, CONTEXT?
```

Comment:
I think you really need to translate this [see](https://github.com/OpenEmu/OpenEmu/blob/cee76cbd2b10b663eb775175ba62332022bb4f99/OpenEmu/it.lproj/InfoPlist.strings#L35)